src/server/utils/api.ts

```diff
@@ -82,6 +82,24 @@ API.get('/portfolio/:userName', async (_req, res) => {
   }
 });
 
+//consulta de la Api
+API.get('/portfolios', async (_req, res) => {

```

Comment:
This endpoint is unnecessary, will get exponentially expensive while the project grows and more users registers

should be removed